src/invidious/config.cr

```diff
@@ -84,6 +84,7 @@ class Config
 
   # Used to tell Invidious it is behind a proxy, so links to resources should be https://
   property https_only : Bool?
+  property login_only : Bool?
```

Comment:
```suggestion
 
   # Only allow usage of the instance with an account
  property login_only : Bool?
  # Redirect to the login page when login_only: true otherwise a 401 status code is returned.
  property redirect_login_only : Bool?
  
```

This allows to choose whether a non whitelisted request will be forwarded to `/login` or not.